ui/component/or-dashboard-builder/src/widgets/or-image-widget.ts

```diff
@@ -0,0 +1,366 @@
+            pos = (propPos / 100) * maxSize;
+        }
+        return pos;
+    }
+
+    private handleContainerSizing(config: ImageWidgetConfig){
+        this.updateComplete.then(() => {
+            this.resizeObserver = new ResizeObserver(debounce((entries: ResizeObserverEntry[]) => {
+                const size = entries[0].contentRect;
+                this.imageSize = {
+                    width: size.width,
+                    height: size.height
+                }
+                this.updateComplete.then(() => {
+                    console.log(this.imageSize);
+                });
+            }, 200))
+            this.resizeObserver.observe(this._imgSize);
+        })
+    }
+
+    private handleMarkerPlacement(config: ImageWidgetConfig) {
+        var xMax = this.imageSize?.width;
+        var yMax = this.imageSize?.height;
+
+        if (this.assetAttributes && config.attributeRefs.length > 0) {
+
+            return this.assetAttributes.map((attribute) => 
+            (
+                html`
+                <span id="overlay" style="
+                left: ${
+                    this.getProportionalPosition(config.xCoordinatesMap[this.assetAttributes.indexOf(attribute)], xMax!)
+                }px;
+                top: ${
+                    this.getProportionalPosition(config.yCoordinatesMap[this.assetAttributes.indexOf(attribute)], yMax!)
+                }px;
+                ">${attribute[1].value}</span>
+                `
+            ));
+        }
+    }
+
+    render() {
+
+        const css =
+            `
+            #img-container {
+                height: 100%;
+                display: flex;
+                justify-content: center; 
+                align-items: center;
+                position: relative;
+                overflow: hidden;
+                z-index: 1;
+            }
+
+            .img-content {
+                display: flex;
+                flex-direction: column;
+                position: absolute;    /*added to check if elements can stack*/
+                height: 100%;
+                width: 100%;
+                object-fit: contain;
+                flex: 1;
+                z-index: 2;
+            }
+
+            /*overlay element doesnt have to have be span, div works too, OG try was with span and it worked*/
+            #overlay {
+                position: absolute;     /*prevously relative*/
+                z-index: 3;
+
+                /*additional marker styling*/
+                color: var(--or-app-color2, ${unsafeCSS(DefaultColor2)});
+                background-color: var(--or-app-color3, ${unsafeCSS(DefaultColor3)});
+                border-radius: 15px;
+                padding: 3px 8px 5px 8px;
+                object-fit: contain;
+                text-overflow: ellipsis;
+            }
+        `
+        var imagePath = this.widget?.widgetConfig.imagePath;
+        return html
+            `
+                <style>
+                    ${css}
```

Comment:
Regarding styling it's better to use Lit's notation for this, so you don't have to introduce a `<style>` tag.
Docs: https://lit.dev/docs/components/styles/

Similar to what you did on `OrImageWidgetSettings` and `OrImageMap`. 😄